resip/src/main/java/fr/gouv/vitam/tools/resip/app/ResipApp.java

```diff
@@ -209,14 +222,10 @@ private static Options createOptions() {
      * The main method for both command and graphic version.
      *
      * @param args the arguments
-     * @throws ClassNotFoundException          the class not found exception
-     * @throws InstantiationException          the instantiation exception
-     * @throws IllegalAccessException          the illegal access exception
-     * @throws UnsupportedLookAndFeelException the unsupported look and feel                                         exception
-     * @throws ResipException                  the resip exception
+     * @throws ResipException the resip exception
      */
-    public static void main(String[] args) throws ClassNotFoundException, InstantiationException,
-            IllegalAccessException, UnsupportedLookAndFeelException, ResipException {
+    public static void main(String[] args) throws
```

Comment:
Pourquoi ne plus lever les exceptions "ClassNotFoundException, InstantiationException,
            IllegalAccessException, UnsupportedLookAndFeelException" ?

Reply:
it's is a code cleaning.